res/data/data.js

```diff
@@ -195,6 +195,38 @@ let json = {
 			"picture":"/res/img/places/Tower_London.jpg",
 			"isVisited":"No", "isLiked":"No"
 		},
+		
+		"thetowerhotel":
+		{
+			"name":"The Tower Hotel",
+			"lat":51.5063443, "long":-0.0753363,
+			"mood":{"tired": 1},
+			"description":"Nestled amidst the River Thames, the Tower of London and St Katharine Docks, The Tower Hotel boasts the finest views in London. No other hotel brings you closer to some of the country’s best loved landmarks.",
+			"weekdays":"-",
+			"weekends":"-",
+			"price":"92.00 - 309.00",
+			"picture":"/res/img/places/Tower_Hotel.jpg",
+			"isVisited":"No", "isLiked":"No",
+		},
+	}],
+	
+	"north":
+	[{
+
+		"templeofseitan":
+		{
+			"name":"Temple of Seitan",
+			"lat":51.537800, "long":-0.130700,
+			"mood":{"":},
```

Comment:
`{"":}`, will break everything. 
Did you mean `{}`?

If so, why not adding a mood before committing?

Reply:
```suggestion
			"mood":{"hungry":8},
```